### PR TITLE
Added Patient Magt Initial AG Grid

### PR DESCRIPTION
**Pull request by BorisLaboratory**
Will be pushing more updates....this is a quick push for any one whose tasks depends on the AG Grid to start  work...

### CODE REVIEW

File: src/pages/Patients.jsx [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+import "ag-grid-community/styles/ag-theme-alpine.css";
 const Patients = () => {
-  return <div>Patients</div>;
+  const [rowData] = useState([
+    {
+      serialNo: 1,
+      patientID: "CHCP1045",
+      patientName: "GERALD ARNORLD",
+      patientEmail: "epipat.pe@gmail.com(CHCC0008)",
+      clinitianAssignedId: "CHCC0008",
```
Comment (dellydel) [inline src/pages/Patients.jsx:12]: spelling

File: src/pages/Patients.jsx [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+    {
+      field: "patientName",
+      valueFormatter: (params) =>
+        params.value.replace(/\w\S*/g, (t) => {
+          return t.charAt(0).toUpperCase() + t.substr(1).toLowerCase();
+        }),
+      headerName: "Patient Name",
+    },
+    { field: "patientEmail", headerName: "Patient Email" },
+    { field: "clinitianAssignedId", headerName: "Clinitian Assigned (ID)" },
```
Comment (dellydel) [inline src/pages/Patients.jsx:30]: spelling

**Review by dellydel — CHANGES_REQUESTED** [review]
small spelling changes, looks good

File: src/pages/Patients.jsx (hunk 2 above)
Comment (BorisLaboratory) [reply src/pages/Patients.jsx:30]: please specify

Comment (dellydel) [reply src/pages/Patients.jsx:30]: clinicians

**Review by dellydel — APPROVED** [review]

**Pull request merged**